pkg/autoscaler/controller/linearcontroller/linear_controller_test.go

```diff
@@ -43,7 +43,8 @@ func TestControllerParser(t *testing.T) {
 		      "nodesPerReplica": 1,
 		      "min": 1,
 		      "max": 100,
-		      "preventSinglePointFailure": true
+		      "preventSinglePointFailure": true,
+			  "IncludeUnschedulableNodes": true
```

Comment:
remove extra tab

Reply:
Mixed tab and spaces for indentation = 😢